_data/jobs.yml

```diff
@@ -41,7 +41,7 @@
   location: State College, Pennsylvania
   name: ICDS Technical Director - Institute for Computational and Data Sciences (ICDS)
   posted: 2021-11-09
-  url: https://psu.wd1.myworkdayjobs.com/en-US/PSU_Staff/job/University-Park-Campus/ICDS-Technical-Director_REQ_0000020689
+  url: https://psu.wd1.myworkdayjobs.com/en-US/PSU_Staff/job/University-Park-Campus/
```

Comment:
Seems better to link to the specific position though? Did the URL checker get impatient or something?

Reply:
Yeah, it yelled about the link not working. It may have been having a moment - want me to revert?